cluster/saltbase/salt/supervisor/docker-checker.sh

```diff
@@ -35,7 +46,7 @@ echo "waiting 30s for startup"
 sleep 30
 
 while true; do
-  if ! sudo timeout 10 docker version > /dev/null; then
+  if ! sudo timeout 10 docker ps > /dev/null; then
```

Comment:
10 second is not enough for `docker ps` at heavy load based on the past experiences (we changed it to `docker ps` and then back to `docker version` once). We'd need a longer timeout. cc @dchen1107